deploy/melange.yaml.tmpl

```diff
@@ -19,13 +19,26 @@ environment:
       - go
       - nodejs
       - yarn
+      - make
 
 pipeline:
   - runs: |
       set -x
       export DESTDIR="${{targets.destdir}}"
       mkdir -p "${DESTDIR}"
 
+      # Build Helm from PR 13439
+      # to be removed later when latest Helm supports --take-ownership flag
+      git clone --depth 1 https://github.com/helm/helm.git
```

Comment:
should use melange `git-checkout` like this: https://github.com/wolfi-dev/os/blob/9a2344577f49c44a869a12f186cb6a11318fe4fd/checksec.yaml#L25-L34

Reply:
and use a specific commit